build/bazelutil/stamp.sh

```diff
@@ -41,15 +41,6 @@ else
     shift 1
 fi
 
-# Handle build-tag.
-if [ -z "${1+x}" ]
-then
-    BUILD_TAG=$(git describe --tags --dirty --match=v[0-9]* 2> /dev/null || git rev-parse --short HEAD;)
```

Comment:
You must delete the line `BUILD_TAG ${BUILD_TAG-}` from the bottom of the script since we're un-defining this variable.

Reply:
Also remove the `build-tag` argument from the documentation above (line 12)